Neos.Flow/Classes/Package/PackageManager.php

```diff
@@ -380,7 +381,11 @@ public function createPackage($packageKey, array $manifest = [], $packagesPath =
         $manifest = ComposerUtility::writeComposerManifest($packagePath, $packageKey, $manifest);
 
         if ($runComposerRequireForTheCreatedPackage) {
-            exec('composer require ' . $manifest['name'] . ' @dev');
+            $rootManifestFilename = FLOW_PATH_ROOT . 'composer.json';
+            $rootManifestContent = json_decode(file_get_contents($rootManifestFilename));
+            $rootManifestContent = Arrays::arrayMergeRecursiveOverrule($rootManifestContent, ['require' => [$manifest['name'] => '@dev']]);
+            file_put_contents($rootManifestFilename, $rootManifestContent);
+            exec('cd ' . escapeshellarg(FLOW_PATH_ROOT) . ' && composer update');
```

Comment:
if this is still an issue - i would add this to only update whats needed.
```suggestion
            exec('cd ' . escapeshellarg(FLOW_PATH_ROOT) . ' && composer update ' . $manifest['name']);
```